cmake/modules/snippets.cmake

```diff
@@ -94,7 +94,17 @@ function(zephyr_process_snippets)
     ERROR_VARIABLE output
     RESULT_VARIABLE ret)
   if(${ret})
-    message(FATAL_ERROR "${output}")
+    list(JOIN SNIPPET_ROOT "\n    " snippet_root_paths)
+    set(snippet_root_paths "\n    ${snippet_root_paths}")
+
+    if(SYSBUILD)
+      zephyr_get(SYSBUILD_NAME SYSBUILD GLOBAL)
+      set(extra_output "Snippet roots for image ${SYSBUILD_NAME}:${snippet_root_paths}\nNote that snippets will be applied to all images unless prefixed with the image name e.g. `-D${SYSBUILD_NAME}_SNIPPET=...`, local snippet roots for one image are not set for other images in a build")
```

Comment:
The [guidelines](https://docs.zephyrproject.org/latest/contribute/style/cmake.html#general-formatting) specify a max line length of 100.

Reply:
Was fighting with bad looking output, solution was to move output outside of quotes in message function call, fixed